packages/block-editor/src/hooks/color.scss

```diff
@@ -0,0 +1,85 @@
+.color-block-support-panel {
```

Comment:
Where does this style come from, it doesn't seem to follow our style guidelines.

Reply:
The [classname comes from the `BlockSupportsToolsPanel`](https://github.com/WordPress/gutenberg/blob/8675f86f885ca389f3bca39971b22cb897bab61b/packages/block-editor/src/components/inspector-controls/block-support-tools-panel.js#L66) which wraps the grouped inspector controls slots for block supports in a `ToolsPanel`.

I can update the format of this class and the corresponding styles for the different block supports dimensions, border, typography etc.